.github/workflows/external-filter.yml

```diff
@@ -0,0 +1,17 @@
+name: External Trigger Filter
+
+on:
+  pull_request:
+    types: [opened, synchronize, reopened, ready_for_review]
+    branches:
+      - 'master'
+    paths-ignore:
+      - '.github/**'
```

Comment:
Do we want to ignore changes in any other sensitive folders or files?
Maybe `bin/**`?